pages/docs/docs.md

```diff
@@ -1324,6 +1348,21 @@ or use parens like so:
     ):
       doStuff()
 
+### Decimals
+
+Numbers in LightScript must begin and end with a number character. That means they
```

Comment:
Also technically, they're floats not decimals, though I think that distinction isn't really useful here (I'm just nervous about getting people's hopes up about an actual Decimal type)